SUILesson3/SUILesson3/ContentView.swift

```diff
@@ -0,0 +1,207 @@
+//
+//  ContentView.swift
+//  SUILesson3
+//
+//  Created by Григоренко Александр Игоревич on 18.01.2023.
+//
+
+import SwiftUI
+
+/// Представление экрана настроек.
+struct ContentView: View {
+
+    // MARK: - private properties
+    @State private var bluetoothMode = Constants.defaultNumber
+    @State private var currentOperator = Constants.defaultNumber
+    @State private var modemMode = Constants.defaultNumber
+    @State private var isAviaModeEnable = false
+    @State private var isVPNEnable = false
+
+    private var settingsStatus = Constants.activateStatus
+    private var wifiStatus = Constants.wifiChanels
+    private var operatorStatus = Constants.operatorNames
+
+    // MARK: - Public properties
+
+    var body: some View {
+        NavigationView {
+            Form {
+                Section {
+                    setupUserProfileView()
+                    setupAppleRecomendarionView()
+                }
+                Section {
+                    setupUpdateView()
+                }
+                Section {
+                    airModeView
+                    setupWIFIView
+                    setupBluetoothView()
+                    setupMobileView()
+                    setupModemView()
+                    setupVpnView()
+                }
+            }
+            .navigationTitle(Constants.settingTitleName)
+        }
+    }
+
+    // MARK: - private methods
+
+    private func setupUserProfileView() -> some View {
```

Comment:
```suggestion
    private var userProfileView() -> some View {
```
  все как Артур написал выше

Reply:
ниже также везде